crates/ibc/src/core/ics02_client/consensus_state.rs

```diff
@@ -26,7 +26,5 @@ pub trait ConsensusState: Send + Sync {
     /// first converting to the raw type (i.e. the protobuf definition), and then
     /// serializing that.
     ///
```

Comment:
Can we remove this empty docstring line?

Reply:
fixed